src/Utils.php

```diff
@@ -63,10 +68,12 @@ public static function debugResource($value = null)
     {
         if (\is_resource($value)) {
             return $value;
-        } elseif (\defined('STDOUT')) {
+        }
+        if (\defined('STDOUT')) {
             return STDOUT;
         }
 
+        /** @var resource */
```

Comment:
Can we leave this back to PHPStan as here we are hiding a possible error.

Reply:
Fixed in 8a5a2db52a9e56f5df09d4975ee89817ab7c4ed3